src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js

```diff
@@ -57,11 +57,16 @@ const transform = (entity, { ancestors }) => ({
     entity.fieldOperatingStatusMoreInfo,
   ),
   fieldPhoneNumber: getDrupalValue(entity.fieldPhoneNumber),
-  fieldRegionPage:
-    entity.fieldRegionPage[0] &&
-    !ancestors.find(r => r.entity.uuid === entity.fieldRegionPage[0].uuid)
+  /* eslint-disable no-nested-ternary */
+  fieldRegionPage: entity.fieldRegionPage[0]
+    ? !ancestors.find(r => r.entity.uuid === entity.fieldRegionPage[0].uuid)
       ? { entity: entity.fieldRegionPage[0] }
-      : null,
+      : {
+          entity: {
+            title: getDrupalValue(entity.fieldRegionPage[0].title),
+          },
+        }
+    : null,
```

Comment:
Not a whole lot better, but something, I guess.

```suggestion
  fieldRegionPage: entity.fieldRegionPage[0]
    ? {
        entity: !ancestors.find(r => r.entity.uuid === entity.fieldRegionPage[0].uuid)
          ? entity.fieldRegionPage[0]
          : { title: getDrupalValue(entity.fieldRegionPage[0].title }
      }
    : null,
```

Reply:
@Clark-W-Griswold @U-DON Should I change them all this way?

Reply:
If you think it's cleaner, go for it.